service/eventer/github/error.go

```diff
@@ -1,19 +1,32 @@
 package github
 
 import (
-	"github.com/juju/errgo"
+	"github.com/giantswarm/microerror"
 )
 
-var invalidConfigError = errgo.New("invalid config")
+var invalidConfigError = &microerror.Error{
+	Kind: "invalidConfigError",
+}
 
 // IsInvalidConfig asserts invalidConfigError.
 func IsInvalidConfig(err error) bool {
-	return errgo.Cause(err) == invalidConfigError
+	return microerror.Cause(err) == invalidConfigError
+}
+
+var missingHeaderError = &microerror.Error{
+	Kind: "missingHeaderError",
 }
 
-var unexpectedStatusCode = errgo.New("unexpected status code")
+// IsMissingHeaderError asserts missingHeaderError.
+func IsMissingHeaderError(err error) bool {
+	return microerror.Cause(err) == missingHeaderError
+}
```

Comment:
To be removed but I can't make a suggestion for this block :(

Reply:
removed